File: kushki-android/src/main/java/com/kushkipagos/android/KushkiEnvironment.java
package com.kushkipagos.android;

public enum KushkiEnvironment implements Environment {
    TESTING("https://uat.aurusinc.com/kushki/api/v1"),
    STAGING("https://staging.aurusinc.com/kushki/api/v1"),
    PRODUCTION("https://p1.kushkipagos.com/kushki/api/v1");

    private final String url;

    KushkiEnvironment(String url) {
        this.url = url;
    }

    @Override
    public String getUrl() {
        return url;
    }
}

File: kushki-android/src/main/java/com/kushkipagos/android/KushkiEnvironment.kt
package com.kushkipagos.android

enum class KushkiEnvironment(override val url: String) : Environment {
    TESTING("https://api-uat.kushkipagos.com/"),
    QA("https://api-qa.kushkipagos.com/"),
    STAGING("https://api-stg.kushkipagos.com/"),
    PRODUCTION("https://api.kushkipagos.com/"),
    UAT_REGIONAL("https://regional-uat.kushkipagos.com/"),
    PRODUCTION_REGIONAL("https://regional.kushkipagos.com/");

}
